DarklyTests/Models/LDUserModelTest.m

```diff
@@ -26,4 +26,64 @@ -(void)testNewUserSetupProperly {
     XCTAssertNotNil(user.updatedAt);
 }
 
+-(void)testDictionaryValue {
```

Comment:
For this test do we not simply want to create a user, call dictionaryValue on it, then initWithDictionary on it? Then compare the initial and recreated objects?